### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by theshadowake** [KidsDontPlay/PlayerStorage#27]
Hi, great mod, looking to implement it in the upcoming Awakening 2 modpack.

Couple of things that would be great if you could do.

1. Have an option to disable the fluid inventory (I would prefer to keep this separate from player inventory).

2. Have a larger player model icon. Possibly at one side of the player storage area (maybe an option to have this or the original).

**Comment by KidsDontPlay** [comment]
you can hover over the player model icon to enlarge it

**Comment by theshadowake** [comment]
Yep, but it is always looking 'down' and the pop out can be a tad jarring, esp when hovering over by mistake (probably just a personal taste/preference thing). Really well executed mod btw.

**Comment by KidsDontPlay** [comment]
ok, I add a config to disbale the fluid storage.
but I don't know what to do with the player icon. the GUI is already pretty big

**Comment by ghost** [comment]
>     2\. Have a larger player model icon. Possibly at one side of the player storage area (maybe an option to have this or the original). <
Good idea.
